src/leappto/cli.py

```diff
@@ -494,8 +527,8 @@ def _rsync():
                                  .format(rsync_dir, ' '.join(self.target_cfg), self.target_addr)
                     Popen(shlex.split(target_cmd)).wait()
 
-                # temporary, after task with different names for containers should be removed
-                shutil.rmtree(rsync_dir)
+                    # temporary, after task with different names for containers should be removed
+                    shutil.rmtree(rsync_dir)
```

Comment:
ok I am so stupid, of course this line must be changed (or moved before ```os.makedirs```), otherwise it's impossible to mount deleted directories for localhost... so I am ok with merge..